docs/src/main/sphinx/optimizer/statistics.md

```diff
@@ -27,6 +27,14 @@ being used and can also vary by table. For example, the
 Hive connector does not currently provide statistics on data size.
 
 Table statistics can be displayed via the Trino SQL interface using the
-{doc}`/sql/show-stats` command. For the Hive connector, refer to the
-{ref}`Hive connector <hive-analyze>` documentation to learn how to update table
-statistics.
+[](/sql/show-stats) command.
+
+Depending on the connector support, table statistics are potentially updated by
+Trino with data changes managed by Trino. For example, the [Delta Lake
+connector](delta-lake-table-statistics), the [Hive connector](hive-analyze), and
+the [Iceberg connector](iceberg-table-statistics) all support table statistics
+management from Trino. You can also initiate updates manually with the
+[](/sql/analyze) command if other systems manipulate the data without Trino so
+that the table statistics are updated. Other connectors rely on the underlying
+data source to manage table statistics or do not support table statistics use at
+all.
```

Comment:
Here's my attempt to make it more explicit when calling ANALYZE in Trino is required:
```suggestion
management from Trino. You can also make Trino collect the statistics with the
[](/sql/analyze) command. This is needed when other systems manipulate the data without Trino, and statistics tracked by Trino are out of date. Other connectors rely on the underlying data source to manage table statistics or do not support table statistics use at all.
```